### ISSUE TITLE
Unsigned.  Chance of add-on being signed?

### ISSUE THREAD

**Issue by oldmacguy** [spenibus/cors-everywhere-firefox-addon#36]
plugin is disabled by Firefox because "CORS-Everywhere could not be verified for use in Firefox and has been disabled."  Firefox does not allow add-ons to be enabled unless unsigned add-ons are allowed in the configuration.  

Any chance of the add-on gaining that trust credential?

**Comment by spenibus** [comment]
Did you get [CORS Everywhere][1] from [AMO][2] ?

[1]: https://addons.mozilla.org/en-US/firefox/addon/cors-everywhere/
[2]: https://addons.mozilla.org/

**Comment by oldmacguy** [comment]
Checking. I pulled it from AMO this morning; others in our group may have gotten it from GitHub or elsewhere(?).  Gathering all the data and will ping back.

**Comment by oldmacguy** [comment]
No, all users pulled from AMO, and go the same error.  We are in an environment that will disallow non-signed packages.

**Comment by spenibus** [comment]
Quite odd. Signing is part of the review process when publishing on AMO. It's pretty much the only reason the addon was made available that way a couple of years back.

Could the following be relevant ?

[Add-ons disabled or failing to install in Firefox][1]

[1]: https://blog.mozilla.org/addons/2019/05/04/update-regarding-add-ons-in-firefox/

**Comment by oldmacguy** [comment]
I’ll walk through that article, although we initially installed on version 76.0.1.
